cookbooks/aws-parallelcluster-config/recipes/update.rb

```diff
@@ -24,3 +24,18 @@
 include_recipe 'aws-parallelcluster-config::directory_service'
 include_recipe 'aws-parallelcluster-slurm::update' if node['cluster']['scheduler'] == 'slurm'
 include_recipe 'aws-parallelcluster-scheduler-plugin::update' if node['cluster']['scheduler'] == 'plugin'
+
+# Update node package - useful for development purposes only
+if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
+
+  service "supervisord" do
+    action :stop
```

Comment:
why stopping the whole supervisord and not just the demon provided by the node package.
Furthermore, are you sure that you are targeting the right supervisord? AFAIK it should be the one present in the cookbook virtualenv, not the one installed at OS level

Reply:
Done.